core/divesite.c

```diff
@@ -314,7 +314,6 @@ void clear_dive_site(struct dive_site *ds)
 	free(ds->name);
 	free(ds->notes);
 	free(ds->description);
-	free(ds->name);
```

Comment:
Are you sure this should be deleted and not replaced by free(ds->country)?

Reply:
i think one of the later changes had moved `ds->country` to be part of the taxonomy.

Reply:
yep, here is the latest `struct dive_site`:
https://github.com/Subsurface-divelog/subsurface/blob/master/core/divesite.h#L16

Reply:
@atdotde There is no ds->country anymore...